### ISSUE TITLE
Made a flexible SelectRawData function

### ISSUE THREAD

**Issue by kapoorlab** [CSBDeep/CSBDeep#13]
Hi, I modified the RawData.from_folder function with an optional 'NumTrain' field. This allows us to put N number of images in training folder but only use M random number of images for creating the training data. 

This could be useful for benchmarking how many images in the training data lead to satisfactory result by comparing the loss function for multiple trainings with different number of images. 

The code is here: 
[https://github.com/kapoorlab/CAREforCurie/blob/master/HelperFunctions/selectrawdata.py](Code)

https://github.com/kapoorlab/CAREforCurie/blob/master/HelperFunctions/selectrawdata.py

I modified the from_folder function from 

`def from_folder(basepath, source_dirs, target_dir, axes='CZYX', pattern='*.tif*'):`

to 

`def from_folder(basepath, source_dirs, target_dir, axes='CZYX', pattern='*.tif*', NumTrain = None):`

and including these lines for random shuffle of images to be included for creating the training data

 `"Shuffle the images in the folder"
        shuffled = sample(image_names, len(image_names))
        
        if NumTrain is None:
            NumTrain = len(image_names)
        
        "Select some of the shuffled imqges for a smaller training dataset"
        image_names = shuffled[:NumTrain]
        
        NumTrain > 0 or _raise(FileNotFoundError("'target_dir' doesn't exist or didn't find any images in it."))`
